packages/orderbook/test/js_api/removeOrder.test.ts

```
@@ -130,14 +130,9 @@ describe('Rain Orderbook JS API Package Bindgen Tests - Remove Order', async fun
 			.thenReply(200, JSON.stringify({ data: { removeOrders: removeOrders } }));
-		try {
-			const result: SgRemoveOrderWithOrder[] = await getTransactionRemoveOrders(
-				mockServer.url + '/sg1',
-				transaction1.id
-			);
-			assert.equal(result[0].order.id, mockRemoveOrder.order.id);
-		} catch (e) {
-			assert.fail('expected to resolve, but failed' + (e instanceof Error ? e.message : String(e)));
-		}
+
+		const result = await getTransactionRemoveOrders(mockServer.url + '/sg1', transaction1.id);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)